Mathmagician.Tests/IntegerTests.cs

```diff
@@ -0,0 +1,39 @@
+﻿using Microsoft.VisualStudio.TestTools.UnitTesting;
+using Mathmagician.Commands;
+
+namespace Mathmagician.Tests
+{
+    [TestClass]
+    public class IntegerTests
+    {
+        [TestMethod]
+        public void IntegerEnsureICanCreateAnIntstance()
+        {
+            Integer my_int = new Integer();
+
+            Assert.IsNotNull(my_int);
+        }
+
+        [TestMethod]
+        public void IntegerReturningAStringOfInts()
+        {
+            Integer testCommand = new Integer();
+            int[] testNumbers = { 1, 2, 3, 4, 5 };
+            testCommand.Numbers.AddRange(testNumbers);
+
+            Assert.IsTrue(testCommand.NumbersToString() is string);
+        }
+
+        [TestMethod]
+        public void IntegerOutputQuantityMatchesUserInput()
+        {
+            Integer outputInts = new Integer();
+            outputInts.CreateNumbers(6);
+
+            Assert.AreEqual(6, outputInts.Numbers.Count);
+        }
+

```

Comment:
Maybe create a test to see if Integer can create integers? The other tests seem good to me.